py/embedded-server/deephaven_server/__init__.py

```diff
@@ -6,3 +6,5 @@
 from .start_jvm import DEFAULT_JVM_PROPERTIES, DEFAULT_JVM_ARGS, start_jvm
 from .server import Server
 
+from deephaven_internal.jvm import check_py_env
+check_py_env()
```

Comment:
I think this is an _ok_ place to check. The other potential place would be before (or as first part of ) `start_jvm`.

Is it more pythonic to run code during import and error out asap?

Reply:
This is really to go by the 'fail early fail fast' principle.